src/Utils/Utils.php

```diff
@@ -448,4 +454,9 @@ public static function flat(array $array): array
 
         return $flattened;
     }
+
+    public static function flattenWithUnique(array $array): array
+    {
+        return array_unique(self::flat($array), SORT_REGULAR);
```

Comment:
Always try to use `static` when in non-private context. When uploading the plugin to the Shopware Store, Shopware's linter doesn't like this, and throws a warning at us :smile: